.github/workflows/replay-verify.yaml

```diff
@@ -23,11 +23,7 @@ on:
       - ".github/workflows/replay-verify.yaml"
       - "testsuite/replay_verify.py"
   schedule:
-    - cron: "0 22 * * *"
-  push:
-    branches:
-      - aptos-release-v* # the aptos release branches
-      - aptos-node-v*
+    - cron: "0 22 * * 2,4,7"
```

Comment:
https://crontab.guru/#0_22_*_*_2,4,7

this says `7` refering to Sunday is not standard.